api/swagger.yaml

```diff
@@ -1969,6 +1969,9 @@ definitions:
                 example:
                   - "sha256:675532206fbf3030b8458f88d6e26d4eb1577688a25efec97154c94e8b6b4887"
                   - "sha256:e216a057b1cb1efc11f8a268f37ef62083e70b1b38323ba252e25ac88904a7e8"
+          ProtocolScheme:
+            type: "string"
+            example: "some.protocol+v1.0"
```

Comment:
What we have is `moby.plugins.http/v1`. The version is separated by a `/`, not by `+` as in this example. 
Which one is it?